indico/modules/rb_new/client/js/modules/calendar/reducers.js

```diff
@@ -22,23 +22,34 @@ import {requestReducer} from 'indico/utils/redux';
 import {serializeDate} from 'indico/utils/date';
 import * as actions from '../../actions';
 import * as calendarActions from './actions';
+import {filterReducerFactory} from '../../common/filters';
+import {processRoomFilters} from '../../common/roomSearch/reducers';
 
 
 export const initialState = {
-    date: serializeDate(moment()),
     rows: [],
+    roomIds: null
+};
+
+export const initialFilterState = {
```

Comment:
make this an arrow function returning the dict so the current date is used and not the one of the first page load